frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleashItem.tsx

```diff
@@ -96,51 +58,55 @@ export const NewInUnleashItem = ({
         onDismiss();
     };
 
+    const onOpen = () => {
+        onClick();
+        handleTooltipOpen();
+    };
+
     return (
```

Comment:
This is now stupid and is primarily responsible for picking between a dialog and a tooltip and managing that state